src/components/core/PivotTable.tsx

```diff
@@ -322,6 +324,8 @@ export class PivotTableInner extends
         const { columnDefs, rowData } = this.state;
         const { pageSize } = this.props;
 
+        const separators = get(this.props, 'config.separators', undefined);
```

Comment:
, undefined is optional, same as get(this.props, 'config.separators')

Reply:
If we remove `undefined` we get `src/components/core/PivotTable.tsx(368,83): error TS2345: Argument of type '{}' is not assignable to parameter of type 'ISeparators'. Property 'thousand' is missing in type '{}'.` in `separators` parameters in `getMeasureCellFormattedValue` and `getMeasureCellStyle` functions

Reply:
wierd...